<launch>

    <param name="use_sim_time" value="false"/> 

    <node pkg="odom_pkg" type="odometry_crawler_node" name="odometry_crawler_node" output="screen">

    </node>

    <node pkg="rviz" type="rviz" name="rviz" output="screen" args="-d $(find odom_pkg)/launch/rviz.rviz"/>

</launch>